src/Traits/WithLoadConfigurationsFrom.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Orchestra\Testbench\Traits;
+
+use SplFileInfo;
+use Symfony\Component\Finder\Finder;
+use Illuminate\Foundation\Application;
+use Illuminate\Contracts\Foundation\Application as ApplicationContract;
+
+trait WithLoadConfigurationsFrom
+{
+    /**
+     * Load config from one folder up.
+     *
+     * @param Application|ApplicationContract $app
+     * @param string                          $configFolder
+     */
+    protected function loadConfigurationFiles($app, $configFolder)
+    {
+        foreach ($this->getConfigurationFiles($configFolder) as $key => $path) {
+            $config = require $path;
+
+            if ($app['config']->has($key)) {
+                $config = array_replace_recursive($app['config']->get($key), $config);
+            }
```

Comment:
This behave differently from any Laravel default setup. Testbench is and should bootstrap Laravel Application based on default behaviour.